cypress/integration/tests/harvest-register.spec.js

```diff
@@ -60,8 +69,128 @@ describe('Harvest register',function (){
         harvestRegister.reportExistsInTheList(reportDate).next()
             .should('be.visible')
             .and("contain", reportDate)
+    })
 
-})
+    it('should verify added data in the report',function (){
+        harvestRegister.addHarvestReport(penM1
+            ,fishHarvestedValueForPenM1
+            ,averageHarvestWeightValueForM1
+            ,fishPenAfterHarvestValueForM1
+            ,penM1Comment
+            ,penM2
+            ,fishHarvestedValueForPenM2
+            ,averageHarvestWeightValueForM2
+            ,fishPenAfterHarvestValueForM2
+            ,penM2Comment
+            ,successfulToasterPopupMessage)
```

Comment:
Why "comma" at the beginning of the word? 

```suggestion
harvestRegister.addHarvestReport(penM1,
            fishHarvestedValueForPenM1,
            averageHarvestWeightValueForM1, ....
```